packages/docusaurus-theme-classic/src/theme/NavbarItem/DefaultNavbarItem.tsx

```diff
@@ -110,27 +116,27 @@ function NavItemDesktop({
         onKeyDown={(e) => {
           if (e.key === 'Enter') {
             e.preventDefault();
-            toggle(true);
+            setShowDropdown(!showDropdown);
           }
         }}>
         {props.label}
       </NavLink>
-      <ul ref={dropDownMenuRef} className="dropdown__menu">
+      <ul ref={dropdownMenuRef} className="dropdown__menu">
         {items.map(({className: childItemClassName, ...childItemProps}, i) => (
           <li key={i}>
             <NavLink
               onKeyDown={(e) => {
                 if (i === items.length - 1 && e.key === 'Tab') {
                   e.preventDefault();
-                  toggle(false);
-                  
-                  const nextNavbarItem =
-                    dropDownRef.current &&
-                    (dropDownRef.current as HTMLElement).nextElementSibling;
+
+                  setShowDropdown(false);
+
+                  const nextNavbarItem = (dropdownRef.current as HTMLElement)
+                    .nextElementSibling;
 
                   if (nextNavbarItem) {
```

Comment:
Wonder if it is possible to shorten this code down to one-line?
```ts
 const nextNavbarItem = (dropdownRef.current as HTMLElement).nextElementSibling;

if (nextNavbarItem) {
  (nextNavbarItem as HTMLElement).focus();
}
```

I have tried using optional chaining, but I ran into the type error.
Any clue? @slorber

Reply:
Afaik it's annoying but we can't do anything about the types not being the proper ones.

The shortest is probably:

```tsx
((dropDownRef.current as HTMLElement)
                    ?.nextElementSibling as HTMLElement)?.focus();
```

Not much better